static/templates/settings/account-settings.handlebars

```diff
@@ -9,24 +9,6 @@
                 <i class="icon-vector-question-sign change_email_tooltip" {{#unless page_params.realm_email_changes_disabled}}style="display:none" {{/unless}} data-toggle="tooltip"
                 title="{{t 'Changing email addresses has been disabled by your Zulip organization administrators. Contact an administrator for help.' }}"></i>
             </p>
-
-            <div id="change_email_modal" class="modal hide" tabindex="-1" role="dialog"
-                aria-labelledby="change_email_modal_label" aria-hidden="true">
-                <div class="modal-header">
-                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    <h3 id="change_email_modal_label">{{t "Change email" }}</h3>
-                </div>
-                <div class="modal-body">
-                    <div class="input-group email_change_container">
-                        <label for="email">{{t "Email" }}</label>
-                        <input type="text" name="email" value="{{ page_params.email }}" autocomplete="off" />
-                    </div>
-                </div>
-                <div class="modal-footer">
-                    <button id='change_email_button' class="btn btn-success" data-dismiss="modal" aria-hidden="true">{{t "Change" }}</button>
-                    <button class="btn btn-primary" data-dismiss="modal" aria-hidden="true">{{t "Close" }}</button>
-                </div>
-            </div>
```

Comment:
I was going to merge just this commit right now, but realized I can't because it actually breaks the feature; when the content is moved into a new template, a partial should be used to include the new template here in this commit.  I think doing that might have saved you the need to write that new template test.